### DISCUSSION TITLE
so i modified the ko.txt in AppData\Local\osu!\Localisation and i added my language in their, it is modifying game files which idk is it still against the tos, if it is i will remove the ko.txt

### DISCUSSION THREAD

**Discussion by catlomao** [ppy/osu#28543] [Q&A]
![image](https://github.com/ppy/osu/assets/84031819/bd05c99a-1bcd-4dd5-92fa-f3e6847b4724)

**Comment by Stoppedpuma** [comment]
This git repo is for osu! lazer, not stable. Considering this isn't cheating and people already use programs for reading the games memory as well as modifying beatmaps at a file level similarly to how you've done with language then I doubt you'll face any issues for adding your language translation. In the future, you should send an email to [support@ppy.sh](mailto:support@ppy.sh) if you're uncertain about these types of things.
